<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="windowTitle">
   <string>Export &amp; Publishing</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="fileLabel">
       <property name="text">
        <string>File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fileLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="uploadCheckBox">
     <property name="text">
      <string>Upload to FTP site</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>FTP Options:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="pathLabel">
        <property name="text">
         <string>Path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="hostLabel">
        <property name="text">
         <string>Host:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="userLabel">
        <property name="text">
         <string>User:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Passoword:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="hostLineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="portLineEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="pathLineEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="userLineEdit"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="passwordLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>ExportDialog</receiver>
   <slot>slotCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okPushButton</sender>
   <signal>clicked()</signal>
   <receiver>ExportDialog</receiver>
   <slot>slotOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uploadCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>ExportDialog</receiver>
   <slot>slotUpload(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotOk()</slot>
  <slot>slotCancel()</slot>
  <slot>slotUpload(int)</slot>
 </slots>
</ui>
